src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,54 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("size() 메소드_Set의 크기 확인")
+    void setSizeTest() {
+
+        int result = numbers.size();
+        // 결과확인
+        assertThat(result).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```
컬렉션을 쉽게 검증할 수 있도록 지원하고 있습니다 😄